.eslintrc

```diff
@@ -52,14 +52,6 @@
     "unicode-bom": ["error", "never"]
   },
   "overrides": [
-    {
-      "files": [ "build/service-worker.js" ],
-      "rules": {
-        "max-len": "off",
-        "no-console": "off",
-        "no-var": "off"
-      }
-    },
```

Comment:
I'm not sure why we were linting a generated file. With workbox the linting fails horribly but I don't think we care because nobody should be reading it.